README.md

```diff
@@ -32,7 +32,7 @@ We have meetings regularly on Wednesdays at 10:30am EST. Feel free to join!
 nsm-operator community meeting <br>
 Wednesdays at 10:30 – 11:30am Eastern Time <br>
```

Comment:
Every 2 weeks

Reply:
Thanks a lot for testing @LionelJouin ! And good catch. Just fixed it. Still not ideal. We should have a calendar as well. We can discuss that in our next meeting.